FreshRSS/FreshRSS
# Improved: Install process: give more infos

Before:
No information about FreshRSS and the installing version
![grafik](https://user-images.githubusercontent.com/1645099/235003623-6015a714-fe77-4806-bda5-86baa6de9852.png)

After:
Display some information
![grafik](https://user-images.githubusercontent.com/1645099/235003674-12d4e992-44fa-4a8e-a8b4-e736d5754c15.png)



How to test the feature manually:

1. Start the install process
2. see the first step

Pull request checklist:
